applications/plugins/PreassembledMass/PreassembledMass.inl

```diff
@@ -57,7 +57,7 @@ void PreassembledMass< DataTypes >::bwdInit()
     if( massMatrix.rows() != massMatrix.cols() || massMatrix.rows()!=(typename MassMatrix::Index)this->mstate->getMatrixSize() )
     {
         // perform assembly
-        core::MechanicalParams mparams = *core::MechanicalParams::defaultInstance();
+        core::MechanicalParams mparams = *core::GetDefaultMechanicalParamsInstance();
```

Comment:
Should be `sofa::core::mechanicalparams::defaultInstance()` ?

Reply:
No